### PR TITLE
support delete

### PR DESCRIPTION
**Pull request by ariririri**
I add the feature which deletes columns.

This function can confirm the following procedure
1. `./build_run.sh`
2. access `localhost:8000/questions`
3. access `delete` link 

If there is no problem, please merge.

### CODE REVIEW

File: volume/samp_app/app/resources/questions/QuestionsController.jl [hunk 1]
```diff
@@ -10,9 +10,13 @@ end
 
 function create()
   Question(question = @params(:question_precise), author = @params(:question_author)) |> save 
-  redirect_to(:questions) 
 end
 
+function delete(id::Integer)
+  SearchLight.delete(SearchLight.find_one!!(Question, id))
+  redirect_to(:questions) 
+  #html!(:questions, Symbol("questiontop.jl.html"), questions = SearchLight.all(Question))
```
Comment (chaspy) [inline volume/samp_app/app/resources/questions/QuestionsController.jl]: is it need?

File: volume/samp_app/samp_app.jl [hunk 2]
```diff
@@ -12,5 +12,6 @@ using Questions
 # add initial seed data
 # Questions.seed()
 print(SearchLight.Database.query("select * from questions;"))
+print(SearchLight.count(Question))
```
Comment (chaspy) [inline volume/samp_app/samp_app.jl]: What is it for?

File: volume/samp_app/app/resources/questions/QuestionsController.jl [hunk 3]
```diff
@@ -10,9 +10,13 @@ end
 
 function create()
   Question(question = @params(:question_precise), author = @params(:question_author)) |> save 
-  redirect_to(:questions)
```
Comment (chaspy) [inline volume/samp_app/app/resources/questions/QuestionsController.jl]: why is it removed?
I think it is better to redirect to top when created question.

File: volume/samp_app/config/routes.jl [hunk 4]
```diff
@@ -9,4 +9,10 @@ route("/questions", QuestionsController.index, named=:questions)
 route("/questions/new", QuestionsController.new)
 route("/questions/create", QuestionsController.create, 
     method = POST, named = :create_question)
-route("/questions/delete", QuestionsController.delete)
+ 
+route("/questions/delete/:id::Int") do
+  id = @params(:id)
+  QuestionsController.delete(id)
+  #redirect_to(:questions)
```
Comment (chaspy) [inline volume/samp_app/config/routes.jl]: what are these comments for?

**Comment by chaspy** [comment]
@ariririri 
Confirmed `delete` is working fine!
Done my first review!
Please check the comments.

**Comment by ariririri** [comment]
@chaspy 
Thank you for your nice review.
I fixed the following 
1. `print(SearchLight.count(Question))`, that is debug code. I remove this. 
     If I want to leave for safe, I" ll write the intend
1. remove `redirect_to(:questions)` is miss, I add aggain.
1. `#redirect_to(:questions)` is also debug code. I remove above reason.

Please review again.

File: volume/samp_app/app/resources/questions/QuestionsController.jl (hunk 1 above)
Comment (chaspy) [reply volume/samp_app/app/resources/questions/QuestionsController.jl]: for debug

File: volume/samp_app/samp_app.jl (hunk 2 above)
Comment (chaspy) [reply volume/samp_app/samp_app.jl]: for debug

File: volume/samp_app/app/resources/questions/QuestionsController.jl (hunk 3 above)
Comment (chaspy) [reply volume/samp_app/app/resources/questions/QuestionsController.jl]: resumed

File: volume/samp_app/config/routes.jl (hunk 4 above)
Comment (chaspy) [reply volume/samp_app/config/routes.jl]: for debug

**Comment by chaspy** [comment]
Looks good to me 👍

**Pull request merged**